ui/components/homepage.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+import FilterListView from './filters/list-view';
+import Autocompleter from './filters/autocompleter';
+
+export default function Homepage() {
+  const filterControls = [
+    React.createElement(FilterListView, {
+      autocompleter: React.createElement(Autocompleter, {
+        insertParam: 'topics__id__in',
+        lookup: 'topics',
+        pathname: '/requirements',
+      }),
+      key: 'topic',
+    }),
```

Comment:
We'll probably need to tweak this a bit in the next iteration (with agencies) as the Autocompleter currently submits/redirects on the first entry selection. There's a multi-select option, too, which seems appropriate.